### PR TITLE
docs: add hertzsentry docs

### PR DESCRIPTION
**Pull request by BaiZe1998**
#### What type of PR is this?

docs

#### What this PR does / why we need it (en: English/zh: Chinese):

en: add hertzsentry docs
zh: 添加 hertzsentry 文档

### CODE REVIEW

**Comment by li-jin-gou** [comment]
PATL @L2ncE  @justlorain

File: content/zh/docs/hertz/tutorials/basic-feature/middleware/sentry.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+---
+
+Sentry 是一个开源的实时错误监控项目，支持很多平台，包括 Web 前端、服务器端、移动端和游戏端等。Hertz 通过使用中间件 [hertzsentry](https://github.com/hertz-contrib/hertzsentry) ，整合了 [Sentry-Go](https://docs.sentry.io/platforms/go/) 的 SDK。提供了一些统一的接口，帮助用户获得 sentry hub 和报告错误信息。
+
+注意：信息上报功能的实现，依旧是以 Sentry 的 Go SDK 为载体。
+
+这个项目参考了 [fibersentry](https://github.com/gofiber/contrib/tree/main/fibersentry) 的实现。
+
+## 安装
```
Comment (justlorain) [inline content/zh/docs/hertz/tutorials/basic-feature/middleware/sentry.md:15]: 安装和导入合并成一个安装吧，和其他中间件保持一致

File: content/zh/docs/hertz/tutorials/basic-feature/middleware/sentry.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+```go
+import "github.com/hertz-contrib/hertzsentry"
+```
+
+## 示例代码
+
+```go
+package main
+
+import (
```
Comment (justlorain) [inline content/zh/docs/hertz/tutorials/basic-feature/middleware/sentry.md:32]: 两段式

File: content/zh/docs/hertz/tutorials/basic-feature/middleware/sentry.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
+---
+title: "Sentry"
+date: 2022-11-25
+weight: 10
```
Comment (justlorain) [inline content/zh/docs/hertz/tutorials/basic-feature/middleware/sentry.md:4]: 10被secure使用了

Comment (BaiZe1998) [reply content/zh/docs/hertz/tutorials/basic-feature/middleware/sentry.md:4]: i got it, thx

File: content/zh/docs/hertz/tutorials/basic-feature/middleware/sentry.md (hunk 1 above)
Comment (BaiZe1998) [reply content/zh/docs/hertz/tutorials/basic-feature/middleware/sentry.md:15]: i got it, thx

File: content/zh/docs/hertz/tutorials/basic-feature/middleware/sentry.md (hunk 2 above)
Comment (BaiZe1998) [reply content/zh/docs/hertz/tutorials/basic-feature/middleware/sentry.md:32]: i got it, thx

**Review by li-jin-gou — APPROVED** [review]

**Review by justlorain — APPROVED** [review]

**Review by L2ncE — APPROVED** [review]

**Comment by li-jin-gou** [comment]
LGTM

**Pull request merged**